analyzer/libretime_analyzer/status_reporter.py

```diff
@@ -163,7 +161,7 @@ def is_web_server_broken(url):
     caused by Airtime or the webserver itself being broken temporarily.
     """
     try:
-        test_req = requests.get(url, verify=False)
+        test_req = requests.get(url)
```

Comment:
This could cause issues with LibreTime being served from a self-signed certificate as described in the docs - https://libretime.org/docs/getting-started/ssl/#deploying-a-self-signed-certificate

Reply:
Maybe we should not advice deploying self signed certificates ? These days having a free certificate is easy using lets encrypt.

This seem not consistent with the existing api_client that do not ignore the certificate check ?

Reply:
And if we really want to support self signed cert we should use something like https://stackoverflow.com/questions/30405867/how-to-get-python-requests-to-trust-a-self-signed-ssl-certificate using env variables or such.

Reply:
I agree that let'sencrypt is better. Maybe let's drop it and just note that it is a breaking change?